.github/workflows/nightly_tests.yaml

```diff
@@ -115,6 +116,33 @@ jobs:
       if: always()
       run: python xpk.py cluster delete --cluster $PATHWAYS_TPU_CLUSTER_NAME --zone=us-central2-b
 
+  rc-cluster:
+    runs-on: [ubuntu-20.04]
```

Comment:
Could you move to Ubuntu 22.04 please? Some packages eg. kueuectl that we are incorporating into this repo have some issues with Ubuntu 20.04